mdal/frmts/mdal_binary_dat.cpp

```diff
@@ -245,6 +248,44 @@ void MDAL::DriverBinaryDat::load( const std::string &datFile, MDAL::Mesh *mesh,
         groupMax->setName( group->name() + "/Maximums" );
         break;
 
+      case CT_RT_JULIAN:
+        // Reference time
+        if ( readIStat( in, sflg, &istat ) )
+          EXIT_WITH_ERROR( MDAL_Status::Err_UnknownFormat );
+
+        if ( read( in, reinterpret_cast< char * >( &time ), 8 ) )
+          EXIT_WITH_ERROR( MDAL_Status::Err_UnknownFormat );
+
+        referenceTime = static_cast<double>( time );
+        group->setReferenceTime( std::to_string( referenceTime ) );
```

Comment:
I would put the same string as is in ASCII, so `"JULIAN`" + std::to_string( referenceTime )` or something like that